README.md

```diff
@@ -5,7 +5,7 @@
 # Introduction
 
 This package provides a client interface to query [Trino](https://trino.io/)
-a distributed SQL engine. It supports Python 2.7, 3.5, 3.6, and pypy.
+a distributed SQL engine. It supports Python 3.5, 3.6, 3.7, 3.8, 3.9 and pypy.
```

Comment:
This list will grow over time so can we instead say Python >= 3.5 and pypy?

Reply:
Fixed.